api/routes/router.go

```diff
@@ -8,24 +8,22 @@ import (
 
 func SetupRoutes(engine *gin.Engine) {
 	// User
-	engine.POST("/signup", controllers.HandleSignUp)
-	engine.POST("/code_validation", controllers.HandleCodeValidation)
-	engine.POST("/new_code", controllers.HandleNewCodeValidation)
-	engine.POST("/user/password/code", controllers.HandleCodeResetPassword)
-	engine.PUT("/user/password", controllers.HandleResetPassword)
+	engine.POST("/user/signup", controllers.HandleSignUp)
+	engine.POST("/user/validate", controllers.HandleCodeValidation)
+	engine.POST("/user/validate/code", controllers.HandleNewCodeValidation)
 
```

Comment:
Houston 🎙️ I think that we have a problem here. I'm missing the routes related with controllers of HandleCodeResetPassword and HandleResetPassword is that ok? 🤔

Reply:
When i opened this PR the PR #70 wasn't merged yet, so, those endpoints doesn't appear here but are already merged in the development branch, so I think merging this PR shouldn't cause any problem. But let me update this branch just to make sure. Thanks for letting me know @SilviaPabon.